### PR TITLE
Feature/trsv2350/implement datapush to data layer dj html

### PR DESCRIPTION
**Pull request by osimuka**

### CODE REVIEW

File: trade_remedies_public/registration/tests.py [hunk 1]
```diff
@@ -266,9 +266,8 @@ def test_valid_all_optional(self):
         self.assertTrue(form.is_valid())
 
 
-class TestV2BaseRegisterView(TestCase):
+class TestV2BaseRegisterView(SimpleTestCase):
```
Comment (yusufsheiqh) [inline trade_remedies_public/registration/tests.py:269]: Nice! We should be using `SimpleTestCase` moar.

File: trade_remedies_public/pytest.ini [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[pytest]
```
Comment (chris-pettinga) [inline trade_remedies_public/pytest.ini:1]: was this configuration not getting picked up from the pyproject.toml file?

**Review by yusufsheiqh — APPROVED** [review]

File: trade_remedies_public/pytest.ini (hunk 2 above)
Comment (osimuka) [reply trade_remedies_public/pytest.ini:1]: @chris-pettinga, Yes, when you want to test the whole suit locally, it dose'nt pick it up

Comment (yusufsheiqh) [reply trade_remedies_public/pytest.ini:1]: Strange. It picked it up for me.

Comment (osimuka) [reply trade_remedies_public/pytest.ini:1]: @yusufsheiqh are these tests on the public repo that its picking up?

Comment (osimuka) [reply trade_remedies_public/pytest.ini]: ok removed it now, thanks for jumping in the call, I can run the tests locally using Django test mange command

**Pull request merged**